README.md

```diff
@@ -68,6 +68,18 @@ can instead pass one or more filenames in explicitly.
 $ clojure -m depot.outdated.main --update ../my-project/deps.edn
 ```
 
+## Freezing snapshots
+
+Maven versions ending in `-SNAPSHOT` are mutable, a maintainer can publish as many snapshots as they like, all with the same version string. This means that re-running the same code twice might yield different results, if in the meanwhile a new snapshot was released.
```

Comment:
Rather than "mutable" versions, Maven refers to things like SNAPSHOT (and RELEASE and LATEST) as "virtual" versions, in case you want to match their terminology. I think that term is better at conveying that they are not mutable, but rather a point of indirection that must be resolved (kind of analogous to Clojure atoms - identities that point to a stream of immutable values).

Reply:
Thanks Alex, I'll update the wording. Maybe the flag itself could be `--resolve-virtual` and also take RELEASE and LATEST into account.